.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us.
+
+    Note you do not need to provide all the fields given but the more you give the better and easier for us to figure
+    out the issue.
```

Comment:
If your PR is not finished, mark it with [WIP] (this is the one time a prefix IS useful)